content/slack/_index.md

```diff
@@ -1,70 +1,37 @@
 ---
-title: About the project's Slack
+title: PrestaShop Project's Slack chat
 
 menu:
   main:
     parent: 'Resources'
-    name: 'Slack'
+    name: 'Slack chat'
 
 aliases:
   - /slack/readme/
 ---
 
-# About the project's Slack
+# PrestaShop Project's Slack chat
 
-## General presentation
+The project's Slack chat is open to anyone who wants to discuss, help and contribute to the PrestaShop open source project. If you are an active contributor, or want to become one, this is definitely a place to be!
```

Comment:
```suggestion
The project's Slack chat is open to anyone who wants to discuss, help and contribute to the PrestaShop open source project. If you are an active contributor, or want to become one, this is definitely the place to be!
```

Reply:
Just one suggestion in the first paragraph: "the place to be" could be better, [more information here in the Merriam Webster](https://www.merriam-webster.com/dictionary/the%20place%20to%20be)

Reply:
You're absolutely right, no idea how I missed that!